pages/common/virt-sysprep.md

```diff
@@ -0,0 +1,20 @@
+# virt-sysprep
+
+> Reset, unconfigure, or customize a virtual machine image.
+> More information: <https://manned.org/virt-sysprep.1>.
+
+- List all supported operations (enabled operations are indicated with asterisks):
+
+`virt-sysprep --list-operations`
+
+- Run the enabled operations but don't apply the changes:
+
+`virt-sysprep --domain {{vm_name}} --dry-run`
+
+- Run the specified operations instead of the enabled operations:
```

Comment:
```suggestion
- Run only the specified operations:
```
Would that also work?

Reply:
I'm not sure about this one. It wasn't immediately clear to me from the man page that if I specified an option or enabled a customization it turned off the enabled operations. It took a number of readings and stackoverflow postings before I figured it out.

Reply:
But my suggestion is how the command works isn't it?

Reply:
Agreed. I thought my elaboration might help readers avoid the missteps I made.